packages/@o3r/telemetry/src/sender/index.ts

```diff
@@ -55,19 +55,32 @@ export type SendDataFn = (data: MetricData, logger?: { error: (msg: string) => v
 /**
  * Send metric to a Amadeus Log Server
  * @param data Metrics to report
- * @param _logger Optional logger to provide to the function
- * @param _logger.error
+ * @param logger Optional logger to provide to the function
  */
-export const sendData: SendDataFn = async (data: MetricData, _logger?: { error: (msg: string) => void }) => {
-  const message = JSON.stringify(data);
-  const body = JSON.stringify({
-    messages: [{
-      applicationName: 'OTTER',
-      message
-    }]
-  });
-  await fetch('https://uat.digital-logging.saas.amadeus.com/postUILogs', {
-    method: 'POST',
-    body
-  });
+export const sendData: SendDataFn = (data, logger) => {
+  let body!: string;
+  try {
+    const message = JSON.stringify(data);
+    body = JSON.stringify({
+      messages: [{
+        applicationName: 'OTTER',
+        message
+      }]
+    });
+  } catch (e: any) {
+    return Promise.reject(e);
+  }
+
+  setTimeout(() => {
+    void fetch('https://uat.digital-logging.saas.amadeus.com/postUILogs', {
+      method: 'POST',
+      body
+    }).catch((e) => {
+      const err = (e instanceof Error ? e : new Error(e));
+      logger?.error(err.stack || err.toString());
+    });
+  }, 1).unref();
```

Comment:
```suggestion
  }, 0).unref();
```
should work too, right?

Reply:
According to https://nodejs.org/api/timers.html#settimeoutcallback-delay-args `0` will be defaulted to `1`